src/Function/Properties/Inverse.agda

```diff
@@ -48,6 +49,9 @@ isEquivalence = record
   ; trans = Composition.inverse
   }
 
+open module ↔ {ℓ} = IsEquivalence (↔-isEquivalence {ℓ})
+  renaming (refl to ↔refl; sym to ↔sym; trans to ↔trans) public
```

Comment:
These names should be `↔-refl`, `↔-sym`, `↔-trans`?

Reply:
I have already updated them.